packages/core/.storybook/main.js

```diff
@@ -0,0 +1,15 @@
+module.exports = {
+  stories: [
+    '../src/**/*.stories.@(ts|mdx)'
+  ],
+  addons: [
+    {
+      name: '@storybook/addon-docs',
+      options: {
+        transcludeMarkdown: true
```

Comment:
Dit is nodig zodat de Stencil-gegenereerde documentatie wordt geparsed voor weergave in "Docs"-tabblad